src/components/homepage/Homepage.jsx

```diff
@@ -1,8 +1,17 @@
+import { useState, useEffect } from 'react'
+import Movie from '../movie/Movie'
+import useFetch from '../../hooks/useFetch';
+import { MovieContainer } from './Styles/StyleMovieContainer';
+
 const Homepage = () => {
+    const url = 'http://localhost:8001/movies';
+    const { movies, isPending, error } = useFetch(url);
     return (
-        <div className="homepage">
-            <p>This is home page</p>
-        </div>
+        <MovieContainer>
+            <div className="movie-container">
```

Comment:
this div is redundant. `MovieContainer` is already a div. Please remove this and move the required styling to the `MovieContainer` div.

Reply:
Done